Add a check for binfmt_misc

Polynomdivision opened the pull request:
Do not merge (yet).

To solve #1223, I added a check at the start of pmbootstrap, that looks for `/proc/sys/fs/binfmt_misc` and prints a message to the user pointing him to the Wiki, if the path does not exist.

ollieparanoid commented on pmb/__init__.py:
```diff
@@ -39,6 +39,7 @@ def main():
     try:
         # Sanity checks
         other.check_grsec(args)
+        other.check_binfmt_misc(args)
```
We could move it here instead (between the `return` and install command):
https://github.com/postmarketOS/pmbootstrap/blob/3d35b45835c961e64b6399c69a9aab74f5cc0093/pmb/chroot/binfmt.py#L36-L38

That way it only runs when the `binfmt_misc` feature is used (e.g. when cross-compiling most packages, or when doing a installation to a foreign arch). There's a case when the `binfmt_misc` module is not needed: when building packages for the host architecture, or running the QEMU VM with the host arch.

ollieparanoid commented on pmb/helpers/other.py:
```diff
@@ -61,6 +61,25 @@ def check_grsec(args):
+        return
+
+    link = "https://wiki.postmarketos.org/wiki/Troubleshooting#sh:_can.27t_create_.2Fproc.2Fsys.2Ffs.2Fbinfmt_misc.2Fregister:_nonexistent_directory"
+    raise RuntimeError("It appears that your system has not loaded the"
```
How about:
> It appears that your system has not loaded the module 'binfmt_misc'. This is required to run foreign arch programs with QEMU (e.g. armhf on x86_64): `<https://postmarketos.org/binfmt_misc>`

(Just created that shorter URL with https://github.com/postmarketOS/postmarketos.org/commit/9d2769986cf3b12206d1729ea24280c56441958b)

Polynomdivision commented:
@ollieparanoid Well, I think that pmOS is an extremely cool project that I would love to see succeed, so I try to help out, where I (with my mediocre programming skills) can!

In the new commits, I changed up the error message of the exception raised by `check_binfmt_misc(...)` and moved the check away from `pmb/__init__.py` to `pmb/chroot/binfmt.py`.
